packages/lesswrong/components/sequences/SequencesGrid.jsx

```diff
@@ -15,6 +15,10 @@ export const styles = theme => ({
   gridContent: {
     marginLeft: -15,
     marginRight: -24,
+    [theme.breakpoints.down('md')]: {
+      marginLeft: 0,
+      marginRight: 0
```

Comment:
Are you sure this shouldn't be `theme.breakpoints.down('sm')`? I think it looks better with the negative margins, and in the 'md' size range (960-1279px) there aren't any constraints there.